scrapers/RealityKings.yml

```diff
@@ -66,19 +66,29 @@ xPathScrapers:
             with: https://www.realitykings.com
 
   sceneScraper:
+    common:
+      $section: //img[@alt=""][contains(@src,"poster")]/../../.. #img>div>div>section
     scene:
-      Title: //h2[contains(@class, "hzrLcs")]/text()
+      Title: $section//h1/text()|$section//h2/text()
       Date:
-        selector: //div[contains(@class, "jtRRdJ")]/text()
+        selector: //div[text()="Release Date:"]/../text()
         parseDate: January 2, 2006
-      Details: //p[contains(@class, "kLbWwG")]/text()
+      Details: //div[text()="Description:"]/../div[not(@class)]/text()
       Tags:
-        Name: //div[contains(@class, "cepogm")]/a/text()
+        Name: 
+            selector: //div[text()="Categories:"]/../a/text()[1]
       Performers:
-        Name: //div[contains(@class, "dEgjPa")]/span/a/text()
+        Name: $section//a[@href and @title]/text()
```

Comment:
```yaml
Name: //h2/following-sibling::div//a[contains(@href,"/model")]/text()

I was looking at this for the performers.

Reply:
For **twistysnetwork**,**teenslovehugecocks**,**sneakysex** the `//h2/following-sibling::div//a[contains(@href,"/model")]/text()` doesnt work it needs `h1` instead of `h2`.